pkg/server/http/ipfs.go

```diff
@@ -49,13 +49,50 @@ func ipfsHandler(lassie *lassie.Lassie, cfg HttpServerConfig) func(http.Response
 		hasAccept := req.Header.Get("Accept") != ""
 		acceptTypes := strings.Split(req.Header.Get("Accept"), ",")
 		validAccept := false
+		includeDupes := true
 		for _, acceptType := range acceptTypes {
 			typeParts := strings.Split(acceptType, ";")
 			if typeParts[0] == "*/*" || typeParts[0] == "application/*" || typeParts[0] == "application/vnd.ipld.car" {
 				validAccept = true
-				break
+				if typeParts[0] == "application/vnd.ipld.car" {
+					// parse 412 car attributes
+					for _, nextPart := range typeParts[1:] {
+						pair := strings.Split(nextPart, "=")
+						if len(pair) == 2 {
+							attr := strings.TrimSpace(pair[0])
+							value := strings.TrimSpace(pair[1])
+							switch attr {
+							case "dups":
+								switch value {
+								case "y":
+								case "n":
+									includeDupes = false
+								default:
+									// don't accept un expected values
+									validAccept = false
+								}
+							case "version":
+								switch value {
+								case "1":
+								default:
+									// don't accept any version but 1
+									validAccept = false
+								}
+							case "order":
+								// for now, dfs traversal satisfies all known orders
```

Comment:
```suggestion
								switch value {
									case "dfs":
									case "unk":
									default:
										// we only do dfs, which also satisfies unk, future extensions are not yet supported
										validAccept = false
									}
```

Reply:
because I think any future order is going to not be compatible with dfs, so we may as well proactively signal that we don't support <your new fancy thing> yet